detectron/datasets/json_dataset_evaluator.py

```diff
@@ -137,6 +141,10 @@ def evaluate_boxes(
     if json_dataset.name.find('test') == -1:
         coco_eval = _do_detection_eval(json_dataset, res_file, output_dir)
     else:
+        logger.warning(
+			'{} eval ignored as annotations are undisclosed on test: {} ignored'
```

Comment:
Looks like there's extra indentation here.

Reply:
@rbgirshick You are right, this line is tab indented.
I am really sorry about it, I double checked it should be mergeable now.